src/index.js

```diff
@@ -252,20 +252,21 @@ fflags.enabled('onetrust', () => {
     pinterest: /epik/,
     tiktok: /ttclid/,
   };
-  const params = Array.from(new URLSearchParams(window.location.search).keys());
-  Object.entries(networks).forEach(([network, regex]) => {
-    params.filter((param) => regex.test(param)).forEach((param) => sampleRUM('paid', { source: network, target: param }));
-  });
-})();
 
-fflags.enabled('email', () => {
-  const networks = {
+  const emailNetworks = {
     mailchimp: /mc_(c|e)id/,
     marketo: /mkt_tok/,
 
   };
-  const params = Array.from(new URLSearchParams(window.location.search).keys());
-  Object.entries(networks).forEach(([network, regex]) => {
-    params.filter((param) => regex.test(param)).forEach((param) => sampleRUM('email', { source: network, target: param }));
-  });
-});
+
+  const knownAdNetworksSection = matchParams(adNetworks, searchParams).map((network) => `paid=${network}`);
+  const knownEmailNetworksSection = matchParams(emailNetworks, searchParams).map((network) => `email=${network}`);
```

Comment:
```suggestion
  const knownEmailNetworksSection = matchParams(emailNetworks, searchParams).map((network) => `email:${network}`);
```

Reply:
Actually:
```suggestion
  const knownEmailNetworksSection = matchParams(emailNetworks, searchParams).map((network) => `owned:email:${network}`);
```

email is always owned.